serviceToXML/src/main/java/tkhal/service/stringToXMLController/CustomerFactory.java

```diff
@@ -0,0 +1,111 @@
+package tkhal.service.stringToXMLController;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+
+public class CustomerFactory {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CustomerFactory.class);
+
+    public static LinkedList<Customer> create(String message) {
+        LinkedList<Customer> list = new LinkedList<>();
+        String[] records = message.split("\n");
+        for (String record : records) {
+            String symbol = record.substring(record.length()-1);
+            if (symbol.equals(",")) {
+                record += "?";
+            }
+            String[] value = record.split(",");
+            if (value[18].equals("\"")) {
```

Comment:
1. hard code is bad idea
2. your solution is not permanent - what if comma will be as a part of value for column different from 18 ?

Reply:
fixed